azurelinuxagent/ga/update.py

```diff
@@ -737,6 +618,24 @@ def _report_extensions_summary(self, vm_status):
                 logger.warn(msg)
                 add_event(op=WALAEventOperation.GoalState, is_success=False, message=msg)
 
+    def _log_agent_supports_versioning_or_not(self):
```

Comment:
Feature is supported by default but cx can turn it off if GA versioning flag is disabled. 

and also, we introduced this logging when e2e need for it to know feature is supported or not by the agent.

Reply:
(see above comment: the test can check the status that the agent reports rather than the log)

If we want to log that the customer disabled the feature let's add a message specific for that. we have several of those messages during agent initialization at the beginning of the run() method